source/fundamentals/connection/connect.txt

```diff
@@ -30,25 +30,25 @@ connect to a MongoDB deployment. It instructs the driver on how it should
 connect to MongoDB and how it should behave while connected. The following
 example shows each part of the connection URI:
 
-.. figure:: /includes/figures/connection-string-parts.png
+.. figure:: /includes/figures/dns_seedlist_connection_string_parts.png
    :alt: Each part of the connection string
 
-In this example, we use ``mongodb`` for the protocol, which specifies
-the :manual:`Standard Connection String Format </reference/connection-string/#std-label-connections-standard-connection-string-format>`.
-
-If your instance or deployment has a DNS SRV record, you can use the
+In this example, we connect to an Atlas MongoDB deployment that has a 
```

Comment:
Suggestion:
I think "Atlas cluster" would be most recognizable given the prior suggestions.

Reply:
Given the info that Sarah provided above, I think "deployment" covers both clusters and serverless, so I'll stick with that here unless there's a strong preference for something else.